tests/io/iceberg/test_iceberg_writes.py

```diff
@@ -421,10 +421,12 @@ def test_complex_table_write_read(local_catalog, complex_table, partition_spec,
     pa_table, schema = complex_table
     table = local_catalog.create_table("default.test", schema, partition_spec=partition_spec)
     df = daft.from_arrow(pa_table)
+    df.show()
```

Comment:
Whats this for?

Reply:
Ah my bad, this was when I was debugging iceberg related issues. Removing